8.js

```diff
@@ -0,0 +1,168 @@
+const real = new Float32Array(2)
+const imag = new Float32Array(2)
+const audioContext = new AudioContext()
+const oscillator1 = audioContext.createOscillator()
+const oscillator2 = audioContext.createOscillator()
+const oscillator3 = audioContext.createOscillator()
+const oscillator4 = audioContext.createOscillator()
+
+real[0] = 0
+imag[0] = 0
+real[1] = 1
+imag[1] = 0
+real[2] = 0.7
+imag[2] = 0
+real[3] = 0.5
+imag[3] = 0
+real[4] = 0.3
+imag[4] = 0
+real[5] = 0.2
+imag[5] = 0
+real[6] = 0.1
+imag[6] = 0
+
+const wave1 = audioContext.createPeriodicWave(real, imag)
+
+oscillator1.setPeriodicWave(wave1)
+oscillator2.setPeriodicWave(wave1)
+oscillator3.setPeriodicWave(wave1)
+oscillator4.setPeriodicWave(wave1)
+
+const gainNode1 = audioContext.createGain()
+const gainNode2 = audioContext.createGain()
+
+oscillator1.connect(gainNode1)
+oscillator2.connect(gainNode2)
+oscillator3.connect(gainNode2)
+oscillator4.connect(gainNode2)
+
+gainNode1.gain.value = 0.5
+gainNode2.gain.value = 0
+
+gainNode1.connect(audioContext.destination)
+gainNode2.connect(audioContext.destination)
+
+const C4 = 440 * 3 / 5
+const G4 = C4 * 3 / 2
+const F4 = C4 * 4 / 3
+const E4 = C4 * 5 / 4
+const B4 = G4 * 5 / 4
+const A4 = F4 * 5 / 4
+const D4 = G4 * 3 / 4
+const C5 = C4 * 2
+const C3 = C4 / 2
+const G3 = G4 / 2
+const B3 = B4 / 2
+const A3 = A4 / 2
+const F2 = F4 / 4
+const G2 = G3 / 2
+const E3 = E4 / 2
+const A2 = A3 / 2
+const F3 = F4 / 2
+const E2 = E3 / 2
+
+oscillator1.frequency.value = C4
+
+oscillator1.start()
+
+while (audioContext.currentTime < 1) {}
```

Comment:
This repetition makes me think we might be able to refactor into a function that we can call with some meaningful values?  This may also suggest a different way to represent the stuff (notes?) above, if we want to iterate over a set and do this while thing to it...